packages/deployment-service/src/default-header-interceptor.ts

```diff
@@ -0,0 +1,14 @@
+import { CallHandler, ExecutionContext, Injectable, NestInterceptor } from '@nestjs/common'
+import { Observable } from 'rxjs'
+import { Response } from 'express'
+
+@Injectable()
+export class DefaultHeaderInterceptor implements NestInterceptor {
+  intercept(context: ExecutionContext, next: CallHandler): Observable<any> {
+    const response = context.switchToHttp().getResponse<Response>()
+
+    response.setHeader('Access-Control-Allow-Origin', '*')
```

Comment:
Think you can just use the cors package to fix this in Sonarcloud

Reply:
I'd rather do it like this than to add another package